modules/test/playwright/pages/object-web/list-type/ListTypeDefinitionsPage.ts

```diff
@@ -59,7 +82,40 @@ export class ListTypeDefinitionsPage {
 		await this.savePicklistButton.click();
 	}
 
-	async goto() {
-		await this.applicationsMenuPage.goToPicklists();
+	async translatePicklistItem(
+		listTypeDefinitionName: string, 
+		listTypeEntryName: string, 
+		locationCode: string
+	) {
+		await this.page.getByRole('link', {name: listTypeDefinitionName}).click();
+		
+		await this.page.frameLocator('iframe')
+			.getByRole('link', { name: listTypeEntryName })
+			.click();
+
+		await this.savePicklistButton.waitFor({ state: 'visible' });
```

Comment:
i think is better to wait for other element in the page instead of the save button, for example the basic info title